catapult-sdk/src/plugins/lockHash.js

```diff
@@ -35,15 +35,16 @@ const lockHashPlugin = {
 			lock: { type: ModelType.object, schemaName: 'hashLockInfo.lock' }
 		});
 		builder.addSchema('hashLockInfo.lock', {
-			account: ModelType.binary,
-			accountAddress: ModelType.binary,
+			senderPublicKey: ModelType.binary,
+			senderAddress: ModelType.binary,
 			mosaicId: ModelType.uint64,
 			height: ModelType.uint64,
 			hash: ModelType.binary
 		});
 
 		builder.addTransactionSupport(EntityType.hashLock, {
-			mosaic: ModelType.uint64,
+			mosaicId: ModelType.uint64,
+			amount: ModelType.uint64,
```

Comment:
why `amount`? it's not in catbuffer? https://github.com/nemtech/catbuffer/blob/master/schemas/lock_hash/hash_lock.cats

Reply:
`Amount` is in mongoDB. but I think you are right, it's removed.

Reply:
need to add amount to codecs (below) - note that ordering matters since codecs serialize and deserialize binary transaction data (must be put after `mosaic`)

Reply:
Fixed